docs/reference/samples/observability.filters-pipelines-unchanged.yaml

```diff
@@ -28,4 +28,4 @@ spec:
     - name: my-output
```

Comment:
We should consider naming any yaml which is not complete in such a way to indicate its not complete

Reply:
Include "Snippet" in the title, or what I prefer is the ellipses top and bottom?

Reply:
Or do you mean the actual filename?   These should all be complete at this point.